doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -274,6 +274,13 @@ Bluetooth samples
 
     * Fixed an interoperability issue with iOS devices by setting the report IDs of HID input and output reports to zero.
 
+* :ref:`peripheral_fast_pair` sample:
+
+  * Disabled :kconfig:option:`CONFIG_BT_SETTINGS_CCC_LAZY_LOADING` Kconfig option to workaround rebond issue related to NCSDK-22921 Zephyr bug.
```

Comment:
If you want to use the bug ticket identifier, I think you need to define it as a known issue in the documentation.

Once the known issue is defined, I think you can shorten the description in both release notes and prj.conf. I think that the first sentence is enough as a release note entry:

_* Disabled :kconfig:option:`CONFIG_BT_SETTINGS_CCC_LAZY_LOADING` Kconfig option to workaround rebond issue related to NCSDK-22921 Zephyr bug._

The customer should go to the known issues if they want to learn all the details related to the issue.
You can also provide a link to the Zephyr issue in the known issue description.

Reply:
@divipillai What would be the best approach for Zephyr bugs that affects our code. Mentioning it in known issues as @kapi-no described or just expanding on it in release notes? Should I mention in known issues 2 bugs - Zephyr bug (1) and the consequence of it - Fast Pair bug (2)?

Reply:
Adding a known issue is also acceptable for me. I was initially wondering if it's worth to add a known issue, but maybe it's actually more proper way of handling this one.

Reply:
Yeah, I too agree here to add one known issue for NCSDK-22921. For zephyr bug : The link to the upstream zephyr bugs is provided here -  https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/known_issues.html#zephyr-repository-issues . 
, and as @kapi-no mentioned, we could add a single entry to the release note. 

Initially, I didn't realize this could be a Known issue.